app/views/ems_block_storage/new.html.haml

```diff
@@ -0,0 +1,3 @@
+= react('ProviderForm', :redirect => ems_block_storage_path,
+  :kind => 'storage',
+  :title => ui_lookup(:model => 'ManageIQ::Providers::StorageManager'))
```

Comment:
You will also need an `edit.html.haml`

Reply:
I opened PR for edit https://github.com/ManageIQ/manageiq-ui-classic/pull/7354